fetch-curr-commit-id

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env python3
+
+# See https://stackoverflow.com/questions/14989858/get-the-current-git-hash-in-a-python-script
+
+import git
+
+repo = git.Repo(search_parent_directories=True)
+print(f"{repo.head.object.hexsha}"[:9])
```

Comment:
9 is a fairly typical default for `core.abbrev`, but it's really meant to be used like `short_sha = repo.git.rev_parse(sha, short=True)` where `--short` will return the shortest _unique_ abbreviation of __at least__ `core.abbrev` characters.

I'm not really sure of the scaling thresholds, but Linus recommends 12 characters, and I'm not sure it's a great idea to hardcode the truncation. (Even though 9 is almost certainly sufficient for uniqueness in our repo for all time, I'd feel better letting git decide what's unique rather than making an arbitrary cut.)

Reply:
Good points, I'll fix.

Reply:
Fixed.